### ISSUE TITLE
Show different puppy on refresh?

### ISSUE THREAD

**Issue by Abby805** [heyitsolivia/secretpuppies#40]
Because the gif shown depends on the (I assume random) parameter appended to the URL, if I refresh the page I see the same puppy. 

It would be great if the randomness were handled another way, so that if I refresh the page I see a different puppy.

**Comment by heyitsolivia** [comment]
Hm this needs some thought, as I'm struggling with this particular UX problem.

Currently:
- users use the unique URL as permalinks to share the puppies
- users also try refreshing to see a new puppy (that was also the original behavior anyway)

Would it be confusing for someone to refresh a unique URL and get a different puppy with a different URL? Thoughts?

**Comment by Abby805** [comment]
I see your point. Alternately, perhaps you could add a "see new puppy" button of some sort. Not quite as intuitive as refreshing, but if it was visible enough it could still be far easier than going into the url bar and deleting the parameter.

**Comment by heyitsolivia** [comment]
Right—I forgot to mention that pressing `space` will also show a different puppy. It's definitely valuable to have a visual cue to hint at that hidden UI. Part of what I think makes this project fun is that element of surprise.

For the first wave of users who were hitting the site because of [Gizmodo](gizmodo.com/press-the-space-bar-for-a-puppy-gif) and [Kickstarter](https://twitter.com/kickstarter/status/634842440357179392)— they already knew to pressing the spacebar. But for the uninitiated who are just finding the site now,  get a new puppy just isn't that intuitive.

I think generating a new puppy on page refresh is worth exploring anyway though—it'll take advantage of the existing exploratory behavior.

Feel free to have a go at it if you like, or I'll work on it later this week.
